troubleshoot/elasticsearch.md

```diff
@@ -0,0 +1,67 @@
+---
+navigation_title: "Elasticsearch"
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/troubleshooting.html
+---
+
+# Troubleshoot {{es}} [troubleshooting]
+
+This section helps you fix issues with {{es}} deployments.
+
+::::{tip}
+If you're using {{ech}}, you can use AutoOps to monitor your cluster. AutoOps significantly simplifies cluster management with performance recommendations, resource utilization visibility, and real-time issue detection and resolution. For more information, refer to [](/deploy-manage/monitor/autoops.md).
+::::
+
+## General [troubleshooting-general]
+
+* [](/troubleshoot/elasticsearch/fix-common-cluster-issues.md)
+* [Cluster health API](https://www.elastic.co/docs/api/doc/elasticsearch/v8/group/endpoint-health_report)
```

Comment:
Best to remove version from API link unless it's something specific that you need from that version?
```suggestion
* [Cluster health API](https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-health_report)
```